src/main/webapp/services/AssignUserService.js

```diff
@@ -0,0 +1,150 @@
+import axios from 'axios';
+import { getIntegrationBaseUrl } from 'symphony-integration-commons';
+import BaseService from './BaseService';
+
+const assignDialog = require('../templates/assignDialog.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+const unexpectedErrorDialog = require('../templates/unexpectedErrorDialog.hbs');
+const forbiddenDialog = require('../templates/forbiddenDialog.hbs');
+const notFoundDialog = require('../templates/issueNotFoundDialog.hbs');
+const successDialog = require('../templates/successDialog.hbs');
+
+export default class AssignUserService extends BaseService {
+  constructor(serviceName) {
+    super(serviceName);
+    this.baseUrl = getIntegrationBaseUrl();
+    this.selectedUser = {};
+  }
+
+  successDialog(issueKey) {
+    this.closeDialog('assignIssue');
+
+    const template = successDialog({
+      issueKey,
+      prettyName: this.selectedUser.prettyName,
+    });
+
+    this.openDialog('userAssigned', template, {});
+  }
+
+  openAssignDialog(data, serviceName) {
+    const dialogsService = SYMPHONY.services.subscribe('dialogs');
+
+    const template = assignDialog({
+      url: data.entity.issue.url,
+      key: data.entity.issue.key,
+      name: data.entity.issue.assignee.displayName,
+    });
+
+    const userData = {
```

Comment:
Same as [issueStateEnricher.js](https://github.com/symphonyoss/App-Integrations-Jira/pull/87/files#diff-7ed864899feb253aa3c951c52ec56565) regarding the object creation repetition